Add clarification in Atoms section in Unicode syntax page (#10725)
elixir-lang/elixir: lib/elixir/pages/unicode-syntax.md

  * `<Start>` additionally includes the code point `_` (005F)
  * `<Continue>` additionally includes the code point `@` (0040)

> Note atoms can also be quoted, which allows any characters, such as `:"hello elixir"`. All Elixir operators are also valid atoms (`:+`, `:@`, `:|>`, etc). The full description of valid atoms is available in the ["Atoms" section in the syntax reference](syntax-reference.html#atoms).

Author: ```suggestion
Note atoms can also be quoted, which allows any characters, such as `:"hello elixir"`. All Elixir operators are also valid atoms (`:+`, `:@`, `:|>`, etc.). The full description of valid atoms is available in the ["Atoms" section in the syntax reference](syntax-reference.html#atoms).
```

  * `<Start>` additionally includes the code point `_` (005F)
  * `<Continue>` additionally includes the code point `@` (0040)

> This document covers only the rules for identifier-based atoms. Atoms can also be quoted, which allows any characters, such as `:'🌢 Elixir'`. The full description of valid atoms is available in the ["Atoms" section in the syntax reference](syntax-reference.html#atoms).
> 
> > Note that all Elixir operators are also valid atoms. Therefore `:+`, `:@`, `:|>`, and others are all valid atoms.

Reviewer: ```suggestion
Note atoms can also be quoted, which allows any characters, such as `:"hello elixir"`. All Elixir operators are also valid atoms (`:+`, `:@`, `:|>`, etc). The full description of valid atoms is available in the ["Atoms" section in the syntax reference](syntax-reference.html#atoms).
```

Author: I commit your suggestion, but I need to correct something first, so please do not merge it yet
